apps/modernization-ui/src/apps/patient/add/extended/inputs/identification/IdentificationView.tsx

```diff
@@ -0,0 +1,16 @@
+import { IdentificationEntry } from 'apps/patient/data/entry';
+import { DataDisplay } from 'design-system/data-display/DataDisplay';
+
+type Props = {
+    entry: IdentificationEntry;
+};
+export const IdentificationView = ({ entry }: Props) => {
+    return (
+        <>
+            <DataDisplay title="Identification as of" value={entry.asOf} required />
```

Comment:
question:  I see that the designs include the '*' in view only but I question if it really should be there.